### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by asdf404**
To translate the component, please pass `messages` property to component.
Where `messages` it's an object with following format:

```js
{
    clear: 'Clear'
}

### CODE REVIEW

File: src/StarRatingInput.js [hunk 1]
```diff
@@ -19,6 +29,8 @@
         },
 
         render: function () {
+            this.props.messages = this.props.messages ? this.props.messages : enMessages();
```
Comment (ikr) [inline src/StarRatingInput.js]: Please use the defaultProps instead

File: README.md [hunk 2]
```diff
@@ -27,3 +27,14 @@ Interaction
...
+## Internationalization
+
+To translate the component, please pass `messages` property to component.
+Where `messages` it's an object with following format:
+
+```js
+{
+    clear: 'Clear'
+}
+```
```
Comment (ikr) [inline README.md]: Please document the dependency on Intl, and explain how it can by poly-filled.

File: package.json [hunk 3]
```diff
@@ -37,7 +37,9 @@
     "url": "https://github.com/ikr/react-star-rating-input/issues"
   },
   "dependencies": {
-    "react": "^0.12.0"
+    "intl": "^0.1.4",
```
Comment (ikr) [inline package.json]: Shouldn't `intl` be a `devDependency`?

File: README.md [hunk 4]
```diff
@@ -27,3 +27,14 @@ Interaction
 
 * `value` -- how many stars are selected
 * `onChange: function (value) {...}` -- your `value` change handler
+
+## Internationalization
+
+To translate the component, please pass `messages` property to component.
```
Comment (ikr) [inline README.md]: I'd remove "to component" here

**Comment by ikr** [comment]
Otherwise, looks good. Please make the noted improvements, and we can publish it all as a v5.0.0

**Comment by asdf404** [comment]
@ikr fixed. Please reopen PR.

**Comment by ikr** [comment]
@asdf404 just published v5.0.0

**Pull request merged**